cmd/convox/templates/init/python/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "2"
+services:
+  web:
+    build: .
+    labels:
+      - convox.port.443.protocol=tls
+      - convox.port.443.proxy=true
+    links:
+      - database
+    ports:
+      - "80:5000"
```

Comment:
Port 5000 is the default, according to the [flask.Flask.run() doc](http://flask.pocoo.org/docs/0.11/api/#flask.Flask.run).

Port 8000, no.

However, this isn't meant to be Flask-specific, only Python-specific, and I believe 5000/8000 for http/https is common for Python apps.